content/announcements/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+
+### Standardization and Industry Adoption
+
+XMPP was submitted to the **Internet Engineering Task Force (IETF)** and officially became a standard in 2004 under the XMPP name with the publication of **RFC 3920 and RFC 3921**.
+
+Several internet service providers across Europe, including **Libertysurf, SAPO, and 1&1**, deployed XMPP-based clients and services. The protocol gained further momentum in 2005 when **Google Talk** adopted XMPP in a federated setup, enabling large-scale messaging interoperability.
+
+In **May 2008**, **Facebook announced the launch of an XMPP interface** for its Chat service, allowing users to connect via **standard desktop XMPP clients**. While Facebook's internal protocol differed, it still provided a [standards-based Jabber/XMPP interface](https://web.archive.org/web/20100318030410/https://developers.facebook.com/news.php?blog=1&story=361). At the time, **Facebook's XMPP chat documentation** was fully available on its developer pages, reinforcing the significance of XMPP in mainstream communication services ([archived documentation](https://web.archive.org/web/20111006161206/https://developers.facebook.com/docs/chat/)).
+
+In **2009**, **Google placed XMPP at the core of its collaborative editing tool, Google Wave**, further demonstrating the protocol’s adaptability beyond instant messaging. Meanwhile, **social networks across the world embraced XMPP**, with notable implementations by **Yandex in Russia** and **StudiVZ in Germany**.
+
+### A Proven Standard for Messaging Interoperability
+
+XMPP has played a crucial role in shaping the modern messaging landscape. Its success demonstrates that **true interoperability is achievable through open, federated protocols**, and that major technology companies—including Google and Facebook—have previously recognized its value. As the DMA now mandates interoperability for messaging services, XMPP remains the most viable and battle-tested solution to meet these requirements.
+
+## Smartphones, WhatsApp, and the Gatekeeping Strategy
+
+Despite XMPP’s success, technical advantages, and suitability for messaging interoperability, its adoption slowed as the messaging landscape evolved. The shift toward proprietary platforms coincided with the rise of smartphones, which led to growing financial stakes in messaging services and increased efforts by dominant players to consolidate control over their networks.
+
+### The Shift to Mobile and the Rise of Proprietary Networks
+
+In **2007**, Apple launched the iPhone, ushering in the smartphone era and the widespread adoption of mobile internet. By **2009**, the introduction of both the App Store and push notification services paved the way for the launch of **WhatsApp**. Messaging, which had previously been limited to SMS on mobile, quickly became a **rich and ubiquitous communication tool**.
```

Comment:
Didn't imessage start off on XMPP?

Reply:
I _think_ I remember @matt-tucker or @gdombiak point out that Jive Software was supporting Apple, back when I visited ... many, many, _many_ moons ago. That might've been imessage, but I'm not sure at all.

Reply:
I've never heard of iMessage using XMPP, but it's not impossible, given its ties to Apple's push notification infrastructure which did use XMPP at some point (and still runs on port 5223).

iChat, on the other hand, was Apple's primary chat client for a long time, and was XMPP-based.

Reply:
I probably confused the two. I don't think iMessage even existed when I made that visit.

Reply:
Oh, yeah, iChat. That'll be the one I was thinking of.